lib/Installation/Partitioner/FormattingOptionsPage.pm

```diff
@@ -82,10 +82,13 @@ sub select_partition_id {
 }
 
 sub select_filesystem {
-    my ($self, $filesystem) = @_;
+    my ($self, $filesystem, $skip) = @_;
+    return if $skip;
     assert_screen(FORMATTING_OPTIONS_PAGE);
     send_key($self->{filesystem_shortcut});
-    send_key 'end';
+    wait_screen_change(sub {
+            send_key 'end';
+    }, 20);
```

Comment:
this is not needed with the current code but i am leaving it as it can be handy in other cases